packages/office-ui-fabric-react/src/components/Nav/examples/Nav.Basic.Example.tsx

```diff
@@ -39,34 +40,24 @@ export class NavBasicExample extends React.Component<any, any> {
                 {
                   name: 'Edit',
```

Comment:
just to be picky then, we probably shouldn't call these items edit and delete....They aren't actions, they are links to other resources.

I could see a great example being external links being handled differently than internal links. That's a great usecase for onLinkClick